corehq/apps/events/tasks.py

```diff
@@ -40,6 +40,17 @@ def sync_mobile_worker_attendees(domain_name, user_id):
         reopen_cases(closed_cases)
 
 
+@task(queue='background_queue', ignore_result=True)
+def close_mobile_worker_attendee_cases(domain_name):
```

Comment:
I just realized this is wrong. I'm fetching all attendee cases and closing them. I should only be fetching cases for mobile workers. I'll update